introduction.md

```diff
@@ -0,0 +1 @@
+Hi. Why do I have to do it? I know these parts of course but maybe there's something interesting later. I can't skip this step and waste my time.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.